recipe/meta.yaml

```diff
@@ -41,4 +41,3 @@ extra:
     - sannykr
     - damianavila
     - parmentelat
-    - Juanlu001
```

Comment:
If Juanlu001 changed this GH handle we can update it later but we cannot leave an unused one due to security risks.